tests/compute/test_new_update_all_hetero.py

```diff
@@ -58,52 +57,66 @@ def _test(mfunc, rfunc):
 
         with F.record_grad():
             g.multi_update_all(
-                {'plays' : (mfunc('h', 'm'), rfunc('m', 'y')),
-                'follows': (mfunc('h', 'm'), rfunc('m', 'y')),
-                'develops': (mfunc('h', 'm'), rfunc('m', 'y')),
-                'wishes': (mfunc('h', 'm'), rfunc('m', 'y'))},
-                'sum')
+                {'follows': (mfunc('h', 'm'), rfunc('m', 'y')),
+                'plays' : (mfunc('h', 'm'), rfunc('m', 'y')),
+                'wishes': (mfunc('h', 'm'), rfunc('m', 'y')),
+                'develops': (mfunc('h', 'm'), rfunc('m', 'y'))},
```

Comment:
Is it possible to generate a more complicated graph with randomized metagraphs?  This is a very important functionality (especially with backward computation which is very hard to debug from Python) and we would like to cover as many cases as possible.

Reply:
Added complexity to unittest.